src/convertToObject.js

```diff
@@ -12,7 +12,12 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+  const newString = sourceString.split(';\n').filter(a => a.includes(':'));
```

Comment:
it is always a good idea to choose meaningful names. It will make your code more readable. 'a' or 'b' is not the best  option